### DISCUSSION TITLE
Your trial of Copilot has ended. Please purchase to continue to use Copilot.

### DISCUSSION THREAD

**Discussion by AsadSumbul** [community/community#113097] [GitHub Education]
### Select Topic Area

Question

### Body

I signed up for the student developer pack, before it, I tried the copilot trial, but now I am unable to redeem my student benefit. Please allow me to redeem it. Thanks

**Comment by XiaoHaoJun** [comment]
same

**Comment by wsqy** [comment]
same

**Comment by ryanhaticus** [comment]
same

**Comment by skostousov** [comment]
same

**Comment by MouhcineDOTNET** [comment]
same

**Comment by CedJeanPhi1** [comment]
Same here
